package.json

```diff
@@ -23,15 +23,14 @@
   },
   "lint-staged": {
     "*.{js,ts,tsx}": [
-      "eslint --fix",
-      "git add"
```

Comment:
Is this now done by default?

Reply:
Yes, https://github.com/okonet/lint-staged/releases?page=7#:~:text=Prior%20to%20version,configuration%20after%20upgrading